$OpenBSD: patch-kcm_serverconfigmodule_cpp,v 1.2 2014/07/09 20:03:30 zhuk Exp $
Allow to choose SQLite driver shipped with Qt itself, it behaves better.
--- kcm/serverconfigmodule.cpp.orig	Thu May  8 12:42:54 2014
+++ kcm/serverconfigmodule.cpp	Sun May 18 21:04:54 2014
@@ -57,6 +57,7 @@ ServerConfigModule::ServerConfigModule( QWidget * pare
   ui_driver.driverBox->addItem( QLatin1String("Mysql"), QVariant( QLatin1String("QMYSQL") ) );
   ui_driver.driverBox->addItem( QLatin1String("PostgreSQL"), QVariant( QLatin1String("QPSQL") ) );
   ui_driver.driverBox->addItem( QLatin1String("SQLite"), QVariant( QLatin1String("QSQLITE3") ) );
+  ui_driver.driverBox->addItem( QLatin1String("SQLite (original)"), QVariant( QLatin1String("QSQLITE") ) );
 
   setButtons( KCModule::Default | KCModule::Apply );
 
